bucket/plex-mpv-shim.json

```
@@ -0,0 +1,21 @@
+{
+    "version": "1.10.3",
+    "description": "Cast media from Plex Mobile and Web apps to MPV. (Unofficial)",
+    "homepage": "https://github.com/iwalton3/plex-mpv-shim",
+    "license": "MIT",
+    "url": "https://github.com/iwalton3/plex-mpv-shim/releases/download/v1.10.3/plex-mpv-shim_1.10.3_installer.exe",
+    "hash": "506e787e3dfe43a812eee79362ceae3a3587b239822612df59a67c1e838ee94f",
```

```suggestion
    "architecture": {
        "64bit": {
            "url": "https://github.com/iwalton3/plex-mpv-shim/releases/download/v1.10.3/plex-mpv-shim_1.10.3_installer.exe",
            "hash": "506e787e3dfe43a812eee79362ceae3a3587b239822612df59a67c1e838ee94f"
        }
    },
```

The app is built in 64-bit.
```
> pelook "C:\scoop dir\apps\plex-mpv-shim\current\run.exe"
loaded "C:\scoop dir\apps\plex-mpv-shim\current\run.exe" / 2588498 (0x277F52) bytes
signature/type:       PE64 EXE image for amd64
image checksum:       0x00284E8E (OK)
machine:              0x8664 (amd64)
subsystem:            2 (Windows GUI)
minimum os:           5.2 (WS03)
linkver:              Microsoft LINK 14.29
```